packages/backend/src/api/endpoints/OrganizationRoleApi.ts

```diff
@@ -0,0 +1,98 @@
+import { joinPaths } from '../../util/path';
+import type { DeletedObject } from '../resources';
+import type { OrganizationRole } from '../resources/OrganizationRole';
+import { AbstractAPI } from './AbstractApi';
+
+const basePath = '/organizations_roles';
+
+type GetOrganizationRoleListParams = {
+  limit?: number;
+  offset?: number;
+};
+
+type CreateParams = {
+  name: string;
+  key: string;
+  description: string;
+  permissions: string[];
+};
+
+type GetOrganizationRoleParams = { roleId: string };
+
+type UpdateParams = {
+  name?: string;
+  description?: string;
+  //TODO(@pantelis): Will this override or union ?
+  permissions?: string[];
```

Comment:
@chanioxaris when updating permissions, is it an addition or override ?

Reply:
It's an override. For additions, you have to use the dedicated endpoint

Reply:
```ts
{ permissions: undefined } // will not override
{ permissions: [] } // will assign no perms to a role
```
@chanioxaris is the above correct ?

Reply:
That's correct!